streaming/java/streaming-state/src/main/java/org/ray/streaming/state/keystate/KeyGroupAssignment.java

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
I'm not sure if we are under Apache LICENSE, if so, it is quite inconsistent that some of our files don't have this LICENSE head. It would cause legal risk.

Reply:
Thanks for risk warning, I'll add the this  LICENSE head for other files